analyses/democracy_index_confounding.Rmd

```diff
@@ -0,0 +1,56 @@
+---
+title: "democracy_index_confounding"
+output:
+  pdf_document:
+    latex_engine: xelatex
+  html_document:
+    df_print: paged
+---
+
+```{r setup, include=FALSE}
+knitr::opts_chunk$set(echo = TRUE)
+```
+
+## Analysis of democracy index as a confounding variable
+
+Start by loading the six month data:
+
+```{r}
+data <- read.csv(file = '../prepped_data/sixmonth.csv')
+democracy <- read.csv(file = '../intermediate_data/democracy_index.csv')
+democracy <- democracy[c("X2019", "country_code")]
+colnames(democracy)[1] <- "democracy_index"
+merged <- merge(data, democracy, by=c('country_code'), all.x=TRUE, all.y=FALSE)
+```
+
+### Regressions on democracy index
+
+```{r}
+summary(lm(formula = casepc ~ democracy_index, data = merged))
+plot(merged$democracy_index, merged$casepc, pch=19)
+```
+
+```{r}
+summary(lm(formula = deathpc ~ democracy_index, data = merged))
+plot(merged$democracy_index, merged$deathpc, pch=19)
+```
+
+```{r}
+summary(lm(formula = cfratio ~ democracy_index, data = merged))
+plot(merged$democracy_index, merged$cfratio, pch=19)
+```
+
+From these plots, it appears that democracy index isn't a good predictor of
+poor Covid-19 outcomes, but when you look at deaths-per-capita, there is a trend
+where the highest deaths-per-capita are in high democracy index countries.
+
+### Regressions on GHSI subcomponent scores and democracy index
+
+```{r}
+summary(lm(formula = casepc ~ prev_emergence_pathogens + early_detection + rapid_response + robust_health_sector + commitments + risk_environment, data = merged))

```

Comment:
How do the numbers change if you just use 'overall' instead of each of the sub-component scores? I'm sure you'll get a better adjusted R-squared, but is it also statistically significant?

Reply:
When I use overall instead of the sub-component scores, the R-squared is super low and the democracy index is not statistically significant. I think that is mostly because democracy_index alone doesn't do a great job of predicting outcomes, but when combined with lots of other factors it can help make a model slightly better.